Pod/Classes/SideMenuManager.swift

```diff
@@ -42,6 +42,12 @@ open class SideMenuManager : NSObject {
      */
     open static var menuPresentMode: MenuPresentMode = .viewSlideOut
     
+    /// Retrieves each already created view controller to front when called. Requires class restoration identifiers. Defaults to false.
+    open static var menuPreserveViewOnPush = false
+    
+    /// Mimics a tab function, by instantly presenting the view controller. Defaults to false.
+    open static var menuTabMode = false
```

Comment:
By setting `animated = false`, hiding the SideMenu via animation is also false, therefore removing any animation of the SideMenu closing itself. The purpose of menuTabMode is to always hide the SideMenu via animation and then instantly present the view controller - a feature that is unavailable by toggling `animated`